rosys/vision/detector_hardware.py

```diff
@@ -89,10 +89,24 @@ async def upload_priority_images():
             rosys.background_tasks.create(upload_priority_images(), name='upload_priority_images')
             self.uploads.priority_queue.clear()
 
-    async def upload(self, image: Image) -> None:
+    async def upload(self, image: Image, tags: list[str] = []) -> None:
```

Comment:
What do you think about making `tags` a keyword-only argument?

Reply:
I like that idea. Please check again